src/convos/response-times.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: response-times
+---
+
+# Response Times
+
+Abbot can automatically notify your [First Responders](xref:frs) when a conversation isn't getting a response quickly enough.
+You can set two different thresholds for your response times:
+
+* Target - The ideal response time. If a customer hasn't received a response in this time, we'll send a notification to the FRs.
+* Deadline - The longest a conversation can go without getting an answer. This could represent a contractual obligation like a Service Level Agreement (SLA).
```

Comment:
```suggestion
* Deadline - The longest a conversation can go without getting a response from your team. This could represent a contractual obligation like a Service Level Agreement (SLA).
```
Most SLAs don't require answers, only responses :)